### PR TITLE
Updated Package xml to Fomat 2

### PR DESCRIPTION
**Pull request by wahlent**
in reference to issue at: https://github.com/ros-industrial/ros_industrial_issues/issues/56

### CODE REVIEW

**Comment by jrgnicho** [comment]
@wahlent can you add the `wrid19` label?

**Comment by gavanderhoorn** [comment]
@jrgnicho: this repository doesn't have it, as it was not on the list for wrid19.

**Comment by gavanderhoorn** [comment]
@jrgnicho @wahlent: I've added the label here as well.

File: staubli/package.xml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0"?>
-<package>
+<package ="2">
```
Comment (gonzalocasas) [inline staubli/package.xml]:
```suggestion
<package format="2">
```

Comment (JeremyZoss) [reply staubli/package.xml]: Was working on a fix while you posted this suggestion (see next commit).  Thanks!

**Review by JeremyZoss — APPROVED** [review]
Small fix required to staubli package's `<package>` tag.  Otherwise, looks good.  Thanks for your contribution!

**Review by gonzalocasas — APPROVED** [review]
lgtm!

**Pull request merged**